modules/flowable-json-converter/src/test/java/org/flowable/editor/language/ServiceTaskConverterTest.java

```diff
@@ -12,20 +12,19 @@
  */
 package org.flowable.editor.language;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertTrue;
-
-import java.util.List;
-
-import org.flowable.bpmn.model.FlowableListener;
 import org.flowable.bpmn.model.BpmnModel;
 import org.flowable.bpmn.model.FieldExtension;
 import org.flowable.bpmn.model.FlowElement;
+import org.flowable.bpmn.model.FlowableListener;
 import org.flowable.bpmn.model.ImplementationType;
+import org.flowable.bpmn.model.MapExceptionEntry;
 import org.flowable.bpmn.model.ServiceTask;
 import org.junit.Test;
 
+import java.util.List;
+
+import static org.junit.Assert.*;
```

Comment:
Looks like most wildcard imports have been corrected but this one at least was missed.
BTW, standards have static imports at the top of the file not at the bottom of the list.

Reply:
Fix on the way